module/zfs/dmu.c

```diff
@@ -2164,6 +2166,11 @@ dmu_write_policy(objset_t *os, dnode_t *dn, int level, int wp, zio_prop_t *zp)
 	} else {
 		compress = zio_compress_select(os->os_spa, dn->dn_compress,
 		    compress);
+#if 0
```

Comment:
Why are we including a code block that is entirely disabled with `#if 0`? Do you feel this will get used at all at some point?

Reply:
The question of if this block of code is needed is still outstanding. In the end it will either be enabled or removed.